src/Bridges/Constraint/Constraint.jl

```diff
@@ -112,6 +113,8 @@ function add_all_bridges(bridged_model, ::Type{T}) where {T}
     MOI.Bridges.add_bridge(bridged_model, IndicatorGreaterToLessThanBridge{T})
     MOI.Bridges.add_bridge(bridged_model, SemiToBinaryBridge{T})
     MOI.Bridges.add_bridge(bridged_model, ZeroOneBridge{T})
+    # Do not add by default
+    # MOI.Bridges.add_bridge(bridged_model, NumberConversionBridge{T})
```

Comment:
I guess a question: should we turn it on automatically?

Reply:
Or should MiniZinc add it as a `ListOfNonStandardBridges`?

Reply:
I think the conclusion is `ListOfNonStandardBridges`: https://github.com/jump-dev/MathOptInterface.jl/issues/1694#issue-1067285249

We can always enable later, but it's breaking to disable.

Reply:
Yes, a solver wanting constraints in a type `S` other than `T` should add the bridge to `S` in `ListOfNonStandardBridges{T}`. Since JuMP only creates constraints of type `T`, I don't know how it would be useful to have the bridge that converts everything to `T` by default. We could wait and see.